File: CVR/batch-4/Pattern1.java
public class Pattern1 {
    public static void main(String[] args) {
        int row = 10;
        int col = 2*row - 1;
        int a = row-1, b = row-1;
        for(int i=0;i<row;i++){
            boolean flag = true;
            for(int j=0;j<col;j++){
                if(j>=a && j<=b){
                    if(flag){
                        System.out.print("*");
                    }else{
                        System.out.print(" ");
                    }
                    flag = !flag;
                }else{
                    System.out.print(" ");
                }
            }
            System.out.println();
            a--;b++;
        }
    }
}

File: CVR/03-07-23/Python/pattern1.py


def printPattern(row):
    col = 2*n-1
    a,b = row -1
    for i in range(row):
        flag = True
        for j in range(col):
            if j>=a and j <=b:
                if flag == True:
                    print("*",end=" ")
                else:
                    print(" ",end=" ")
                flag= not flag
            else:
                print(" ")
        a-=1
        b+=1
        print()


printPattern(10)
            

